common/build.gradle.kts

```diff
@@ -97,3 +97,20 @@ android {
         targetCompatibility = JavaVersion.VERSION_1_8
     }
 }
+
+val packForXcode by tasks.creating(Sync::class) {
+    group = "build"
+
+    //selecting the right configuration for the iOS framework depending on the Xcode environment variables
+    val mode = System.getenv("CONFIGURATION") ?: "DEBUG"
+    val framework = kotlin.targets.getByName<KotlinNativeTarget>("ios").binaries.getFramework(mode)
+
+    inputs.property("mode", mode)
+    dependsOn(framework.linkTask)
+
+    val targetDir = File(buildDir, "xcode-frameworks")
+    from({ framework.outputDirectory })
+    into(targetDir)
+}
+
+tasks.getByName("build").dependsOn(packForXcode)
```

Comment:
![image](https://user-images.githubusercontent.com/1173057/73763075-cb655800-4725-11ea-9cdd-c5581c89adce.png)

Small comment, it's good to end with a newline. 🙂

Reply:
Right. Small thing, I forgot about that. Fixed